go/pkg/client/cas.go

```diff
@@ -1299,7 +1364,8 @@ func (c *Client) downloadSingle(ctx context.Context, dg digest.Digest, reqs map[
 	// The lock is released when all file copies are finished.
 	// We cannot release the lock after each individual file copy, because
 	// the caller might move the file, and we don't have the contents in memory.
-	defer func() { afterDownload([]digest.Digest{dg}, reqs, err) }()
+	bytesMoved := map[digest.Digest]int64{}
+	defer func() { afterDownload([]digest.Digest{dg}, reqs, bytesMoved, err) }()
```

Comment:
not related to this CL but why don't we just call

`defer afterDownload([]digest.Digest{dg}, reqs, bytesMoved, err)()` ?